radicle-cli/tests/commands.rs

```diff
@@ -192,3 +152,23 @@ fn rad_patch() {
     test("examples/rad-issue.md", working.path(), Some(home), []).unwrap();
     test("examples/rad-patch.md", working.path(), Some(home), []).unwrap();
 }
+
+#[test]
+fn rad_clone() {
+    logger::init(log::Level::Debug);
+
+    let mut environment = Environment::new();
+    let mut alice = environment.node("alice");
+    let bob = environment.node("bob");
+    let working = environment.tmp().join("working");
+
+    // Setup a test project.
+    let _ = alice.project("acme", "");
+
+    let alice = alice.spawn(Config::default());
+    let mut bob = bob.spawn(Config::default());
+
+    bob.connect(&alice).converge([&alice]);
+
+    test("examples/rad-clone.md", working, Some(&bob.home), []).unwrap();
```

Comment:
This is better than before :+1:

As a follow up, how about having the full workflow covered in the documentation tests?  Having everything covered in the documentation tests makes it very clear to readers as well as testers, with everything being in one place.

Here Alice's project setup could take place in a documentation test, or part of 'rad-init.md'.

Reply:
Hmm true, although rad-auth won't be covered, so not sure we're gaining much? We're also starting the node in the test and waiting for it to converge..

I think if we want to test everything, we should find a way to test everything; I'm not sure we gain much from having 80% in the doc test vs 50%..

Reply:
> I think if we want to test everything, we should find a way to test everything

I'm unsure about covering all the nuances but :+1: